flight/mcu/lib/EigenArduino-Eigen30/EigenAVR.h

```diff
@@ -39,7 +39,8 @@ namespace std {
 }
 
 // Include Eigen's Core
```

Comment:
nit: not sure what difference eigen/dense makes, but that comment needs to be updated.